bindings/wasm/src/did/wasm_document_metadata.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020-2021 IOTA Stiftung
```

Comment:
Nit: update licence headers to 2022 (we should really just find and replace all of these at some point now, not in this PR though).